batch/batches/Cleanup/KAsyncDirectoryCleanup.class.php

```diff
@@ -12,6 +12,8 @@
  */
 class KAsyncDirectoryCleanup extends KPeriodicWorker
 {
+	const CHUNK_ENCODEING_POSTFIX = 'chunkenc';
```

Comment:
You should get it from  infra/chunkedEncode/KChunkedEncode.php 
By adding const to this class.